docs/userman/gui/extension/dynamic_element/list_of_value_props.md

```diff
@@ -0,0 +1,159 @@
+The previous section on [Tabular properties](tabular_data_props.md) explains creating dynamic elements for tabular data values.
```

Comment:
So in my opinion, this section will appear *before* the one on tabular data, since it's far easier to grasp.